mocca-client/build.gradle

```diff
@@ -3,8 +3,12 @@ dependencies {
                    lib.feign_core,
                    lib.jackson_databind,
                    lib.jackson_datatype_jsr301,
-                   lib.jackson_modules_java8
+                   lib.jackson_modules_java8,
+                   lib.jakarta_validation_api
```

Comment:
This dependency here is not made available transitively to the application using the Mocca client.

In order to do so, change this to this:

``` groovy
dependencies {
    implementation lib.slf4j_api,
                   lib.feign_core,
                   lib.jackson_databind,
                   lib.jackson_datatype_jsr301,
                   lib.jackson_modules_java8

    api lib.jakarta_validation_api

    testImplementation lib.testng,
                       lib.slf4j_simple,
                       lib.wiremock,
                       lib.hibernate_validator,
                       lib.jakarta_el
}
```

Reply:
ok